doc/dev/kubernetes.rst

```diff
@@ -0,0 +1,224 @@
+
+.. _kubernetes-dev:
+
+=======================================
+Hacking on Ceph in Kubernetes with Rook
+=======================================
+
+.. warning:
```

Comment:
this warning is really a commented out section, to be an actual warning you need double colons at the end:

    .. warning::